### PR TITLE
fix(common):replace let by const in locales for fr-BE and nl-BE

### PR DESCRIPTION
**Pull request by jestersimpps**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

change let to const in locale files to avoid linting errors

```
[ ] Bugfix
[ ] Feature
[x ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Comment by jestersimpps** [comment]
I signed it!

File: packages/common/locales/fr-BE.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@
 const u = undefined;
 
 function plural(n: number): number {
-  let i = Math.floor(Math.abs(n));
+  const i = Math.floor(Math.abs(n));
```
Comment (alfaproject) [inline packages/common/locales/fr-BE.ts]: Read line 9

File: packages/common/locales/nl-BE.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@
 const u = undefined;
 
 function plural(n: number): number {
-  let i = Math.floor(Math.abs(n)), v = n.toString().replace(/^[^.]*\.?/, '').length;
+  const i = Math.floor(Math.abs(n)), v = n.toString().replace(/^[^.]*\.?/, '').length;
```
Comment (alfaproject) [inline packages/common/locales/nl-BE.ts]: Read line 9

**Review by alfaproject — CHANGES_REQUESTED** [review]

**Comment by trotyl** [comment]
Please respect:

> `// THIS CODE IS GENERATED - DO NOT MODIFY`
> `// See angular/tools/gulp-tasks/cldr/extract.js`

File: packages/common/locales/fr-BE.ts (hunk 1 above)
Comment (jestersimpps) [reply packages/common/locales/fr-BE.ts]: oops

**Pull request closed without merging**